src/kinetics/GasKinetics.cpp

```diff
@@ -184,6 +193,11 @@ void GasKinetics::updateROP()
         processFalloffReactions();
     }
 
+    // reactions involving third body
+    for (auto & index : m_multi_indices) {
```

Comment:
```suggestion
    for (auto& index : m_multi_indices) {
```
(likewise a few lines below)